usb/device/dev_lowlevel/dev_lowlevel.c

```diff
@@ -489,6 +489,9 @@ static void usb_handle_buff_status() {
  *
  */
 /// \tag::isr_setup_packet[]
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
Should this snippet be moved **above** the `/// \tag::isr_setup_packet[]` line? I don't think we want this `#ifdef` stuff appearing in section 4.1.3.2.3. of https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf ?

Reply:
I wasn't aware of what that did. It should be fixed now.

Reply:
Me neither!